lib/geordi/cucumber.rb

```diff
@@ -8,9 +8,17 @@
 
 module Geordi
   class Cucumber
+    def run(cucumber_options, options = {})
```

Comment:
These are not only options, but all arguments to the Cucumber command. Since we're in the Cucumber context, I'd leave out "cucumber_", and I'd generalize the name to "arguments":
```suggestion
    def run(arguments, options = {})
```